trieste/bayesian_optimizer.py

```diff
@@ -500,6 +506,26 @@ def optimize(
                                 np.min(tagged_output[tag].observations),
                                 step=step,
                             )
+                        tf.summary.scalar(
+                            "wallclock.total",
```

Comment:
total sounds like overall time, e.g. time from the start of the optimization and up until this point. Maybe "step_time"?

Reply:
yep